core/src/test/scala/org/apache/spark/util/UtilsSuite.scala

```diff
@@ -35,7 +35,32 @@ import org.apache.hadoop.fs.Path
 import org.apache.spark.SparkConf
 
 class UtilsSuite extends FunSuite with ResetSystemProperties {
-
+  test("timeConversion") {
+    assert(Utils.timeStringAsS("1") === 1)
+    assert(Utils.timeStringAsS("1s") === 1)
+    assert(Utils.timeStringAsS("1000ms") === 1)
+    assert(Utils.timeStringAsS("1000000us") === 1)
+    assert(Utils.timeStringAsS("1min") === 60)
+    assert(Utils.timeStringAsS("1h") === 60 * 60)
+    assert(Utils.timeStringAsS("1d") === 60 * 60 * 24)
+    
+    assert(Utils.timeStringAsMs("1") === 1)
+    assert(Utils.timeStringAsMs("1ms") === 1)
+    assert(Utils.timeStringAsMs("1000us") === 1)
+    assert(Utils.timeStringAsMs("1s") === 1000)
+    assert(Utils.timeStringAsMs("1min") === 1000 * 60)
+    assert(Utils.timeStringAsMs("1h") === 1000 * 60 * 60)
+    assert(Utils.timeStringAsMs("1d") === 1000 * 60 * 60 * 24)
```

Comment:
minor: you could use `TimeUnit.DAYS.toMillis(1)` here, for example (also in other tests).